mongo/errors.go

```diff
@@ -24,17 +24,35 @@ import (
 // ErrClientDisconnected is returned when disconnected Client is used to run an operation.
 var ErrClientDisconnected = errors.New("client is disconnected")
 
+// InvalidArgumentError wraps an invalid argument error.
+type InvalidArgumentError struct {
+	Wrapped error
```

Comment:
I recommend not exporting `Wrapped` to allow us to change how it's implemented in the future without breaking changes.